tensorflow/core/protobuf/rewriter_config.proto

```diff
@@ -107,10 +107,16 @@ message RewriterConfig {
   // Note that this can change the numerical stability of the graph and may
   // require the use of loss scaling to maintain model convergence.
   Toggle auto_mixed_precision = 23;
-  // Optimize data types for MKL (default is OFF).
+  // Optimize data types for oneDNN (default is OFF).
   // This will try to use bfloat16 on CPUs, which is faster.
   // Note that this can change the numerical stability of the graph.
-  Toggle auto_mixed_precision_mkl = 25;
+  // Note: this config is deprecated
```

Comment:
I would mention here that it's been replaced by `auto_mixed_precision_bfloat16` (or actually, per @penpornk's suggestion, `auto_mixed_precision_onednn`).

And for the comment below, instead of "this replaces auto_mixed_precision_mkl", I would say "this is equivalent to the deprecated option auto_mixed_precision_mkl"